components/list/list.js

```diff
@@ -0,0 +1,59 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], position }) {
+  if (position) {
+    mods.push(`position_${position}`);
+  }
+
+  if (hideBullets) {
+    mods.push("no_bullets");
+  }
+
+  const className = getClassNamesWithMods('ui-list', mods);
+
+  const itemsBlock = items.map((item, index) => (
+    <li key={index}>
+      <span className="ui-list__content">{item}</span>
+    </li>
+  ));
+
+  return (
+    <ul className={className}>
+      {itemsBlock}
+    </ul>
+  );
+}
+
+List.defaultProps = {
+  hideBullets: false,
+  position: 'vertical',
+};
+
+List.propTypes = {
+  /**
+   * List's elements.
+   */
+  items: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
well, I can make it like this:
`
items: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.string),
    PropTypes.element,
  ]).isRequired,
`

so we can pass an element, but why do you need to use List component to pass one Text or Button there? or you are going to make an array of components? sorry, but maybe i'm not getting your idea. 

in case if we need to have list's elements wrapped with `a` or `span` we can make a prop `type` where we can specify if it should be a link or something else